File: C/Estruturas Repetitivas/Exercício 4/crescente.c
#include <stdio.h>

int main() {
    int X, Y;

    printf("Digite dois numeros:\n");
    scanf("%d\n", &X);
    scanf("%d", &Y);

    while (X != Y) {
        if (X > Y) {
            printf("DECRESCENTE!\n");
        } else {
            printf("CRESCENTE\n");
        }
        printf("Digite dois numeros:\n");
        scanf("%d\n", &X);
        scanf("%d", &Y);
    }
    return 0;
}

File: Python/Estruturas Repetitivas/crescente.py
X: int; Y: int

print("Digite dois numeros:")
X = int(input())
Y = int(input())

while X != Y:
    if X > Y:
        print("DECRESCENTE!")
    else:
        print("CRESCENTE!")

    print("Digite outros dois numeros:")
    X = int(input())
    Y = int(input())

